serviceregistration/consul/consul_service_registration.go

```diff
@@ -229,6 +232,22 @@ func (c *serviceRegistration) merge(conf map[string]string) error {
 		c.logger.Debug("config service_tags set", "service_tags", tags)
 	}
 
+  // Get user-defined meta tags to attach to the registered service name
```

Comment:
The formatting looks a little strange here -- could you give this a `make fmt`/align this?

Reply:
Yeah, I used github's UI to resolve the refactor of `NewServiceRegistration` / `serverRegistration.merge` and did not really pay attention, while I had hope, I was not really expecting someone to bring this PR back from the dead 😂 . Fixing!